CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+2.0.0-SNAPTHOT
+* Deprecate FaunaClient(secret:domain:scheme:port:timeout:clientIO:) constructor
```

Comment:
We didn't deprecate that constructor. It was removed.
We also need information about the new data structure, the removal of Ref(string) syntax sugar, the API version supported, etc.